Frontend/src/components/StudentInteraction/Interaction.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import Popover from "react-bootstrap/Popover";
+import OverlayTrigger from "react-bootstrap/OverlayTrigger";
+import Button from "react-bootstrap/Button";
+import { ListGroup } from "react-bootstrap";
+import data from "../data/modalMessage.json";
+import InteractionModal from "../InteractionModal";
+import StudentService from "../../services/student.service";
+
+function Interaction(props) {
+  const id = props.id;
+  const modalMessage = data.delete.message;
+
+  const [show, setShow] = useState(false);
+
+  const handleShow = () => {
+    setShow(true);
+  };
+
+  const handleConfirm = () => {
+    StudentService.deleteStudent(id)
```

Comment:
Missing `await` in the context.

If there are errors, they may not be handled nicely.